scripts/removeSourcemaps.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-console */
+const rimraf = require("rimraf");
+
+/*
+  When the sentry is enabled, remove sourcemaps
+*/
+
+if (
+  process.env.SENTRY_ORG &&
+  process.env.SENTRY_PROJECT &&
+  process.env.SENTRY_DSN &&
+  process.env.SENTRY_AUTH_TOKEN
+) {
+  rimraf("./build/**/*.js.map", () => console.log("Source maps removed."));
```

Comment:
**suggestion**: maybe we can document why we are removing `js.maps` when Sentry is turned on for the future reference - e.g in `docs/` folder in repo 🤔

Reply:
hmm I think this file is enough, but I will put more info there